package.json

```diff
@@ -17,9 +17,9 @@
     "file-type": "^10.5.0",
     "joi": "^13.3.0",
     "js-yaml": "^3.12.0",
-    "request": "^2.87.0",
-    "request-promise": "^4.2.2",
-    "rimraf": "^2.6.2"
+    "rimraf": "^2.6.2",
+    "simple-get": "^4.0.0",
```

Comment:
honorary "why not node-fetch" questions apply :p

Reply:
"I don't know what the node landscape is like, but this seemed to work" response :D

Reply:
fair. node-fetch handles a bit more than get for when we get around to it, but simple-get is the same idea. Either is fine, really.